claimedtasks.md

```diff
@@ -57,8 +57,42 @@ This is a **read-only** endpoint, the [POST to the claimed task](#post-method-si
 
 It returns the tasks claimed by the specified user
 
-## POST Method (collection level)
-The creation of claimed tasks is managed by the underline workflow system. No methods are exposed to manually trigger such creation to avoid workflow hjack and inconsistency.
+#### findAllByItem
+**/api/workflow/claimedtasks/search/findAllByItem?uuid=<:item-uuid>**
+It returns all the claimed tasks related to the specified item
```

Comment:
I'd prefer we note as part of the description that this endpoint is ADMIN only.  Otherwise, you have to look at all the error codes in order to realize it.